osbs/repo_utils.py

```diff
@@ -171,6 +170,54 @@ def __init__(self, dir_path='', file_name=REPO_CONFIG_FILE, depth=None,
     def is_autorebuild_enabled(self):
         return self._config_parser.getboolean('autorebuild', 'enabled')
 
+    def _check_and_validate_container_config_file(self, dir_path):
```

Comment:
Seeing `*_and_*` pattern in functions makes me cry. 

`check` can be just single single function used with parameters (eventually 2 partial function made from it)

When checks passes just call another function `validate` after

Reply:
fixed